Calculator/README.md

```diff
@@ -0,0 +1,29 @@
+# Calculator Instructions:
+
+## Buttons.
+
+-Each button has assigned it's respective symbol it uses, simply press the button with the desired symbol and it will appear on the display of the calculator.
+
+### Special Characters.
+
+-The button with the flask is the calculators scientific mode to use more difficult equations and problems.
+
+-The C button stands for "Clear" and it clears the calculator of all characters on display.
+
+-The del button deletes the last character typed.
+
+-The +/- symbol is to turn the last number typed in into it's opposite, so if it is positive it will turn it negative and vise versa.
+
+-The / is for division, the x is for multiplication, the + is for addition, the - is for subtraction and the . represennts the decimal point.
+
+## How to use.
+
+To use the calculator type in the desired equation and when done press "Equals" or "=" to see the result.
+
+## Things to keep in mind.
+
+Dividing by Zero will show "Infinity".
```

Comment:
just out of curiosity, did you intend for this to happen of it ended up happening because of the weird numbers type of js?

Reply:
What I mean by that, is that a design choice?